pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - 'packages/*'
+  - 'packages/vue/*'
```

Comment:
Not as part of this PR but a follow up should move the vue packages up again

Reply:
I created https://github.com/eclipsesource/jsonforms/issues/2158 as a follow up so it doesn't get lost :)